device/daita.go

```diff
@@ -184,7 +184,7 @@ func injectPadding(action Action, peer *Peer) {
 	elem.padding = true
 	elem.machine_id = &action.Machine
 
-	size := action.Payload.ByteCount + DaitaHeaderLen
+	size := action.Payload.ByteCount
```

Comment:
You need to make sure that `size` is not larger than MTU. Also refer to other implementations to check the behavior when `size` is smaller than `DaitaHeaderLen`.

Reply:
Down below, consider whether `binary.BigEndian.PutUint16()` would yield more clear and direct code.

Reply:
Done and done